src/hooks/useSpaceStyle.ts

```diff
@@ -35,19 +41,22 @@ export const useSpaceStyle = (props: SpaceProps): ViewStyle => {
   const paddingRight = theme.rem(typeof right === 'number' ? right : right ? 1 : 0)
 
   // Direction:
-  const { sideways = false } = props
+  const { isSideways: sideways = false } = props
   const flexDirection = sideways ? 'row' : 'column'
 
   // Alignment:
-  const { start = false, center = false, end = false } = props
```

Comment:
I realize one of my motivations for naming things was to keep things really short. Now that I see what the names were `start` `center` and `end`, I recall how it was meant to be used with multiple space components leveraging the `sideways` flag to control the direction. If one wanted to do `justifyContent` they'd need an extra component, but I see the value in not needing the extra component to change the alignment for this direction.

I wonder if there is a set of words we could use that are short that also encode the direction?